pkg/test/common.go

```diff
@@ -15,6 +15,7 @@ import (
 	"log"
 	"math"
 	"math/big"
+	mathRand "math/rand"
```

Comment:
Only the content of the layers is different to randomize the digest of the manifest. In case we need the same image we can set the layers and config to a default value that's constant. That's the `DefaultLayers` and `DefaultConfig`